### DISCUSSION TITLE
how to get page separation details

### DISCUSSION THREAD

**Discussion by murali-m-sgsco** [pymupdf/PyMuPDF#2581] [Looking for help]
I need to get page separation details ( fz_page_separations ) like name and separation RGB color code with pymupdf

also is there any possible to remove a separation ?

**Comment by JorjMcKie** [comment]
PyMuPDF does not currently provide access to page separation details - sorry.

**Reply by murali-m-sgsco** [reply]
so we can disable the separations with fz_page_separations on C++?

**Reply by JorjMcKie** [reply]
I am not exactly sure what you mean. If you are dealing with your own MuPDF source version in that respect, then it should not have implications for PyMuPDF.
